tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -391,6 +391,50 @@ def test_arxiv_plot_extract_is_safe_to_rerun(mock_os):
         rmtree(temporary_dir)
 
 
+@patch('plotextractor.api.os')
+def test_arxiv_plot_extract_handles_duplicate_plot_names(mock_os):
+    schema = load_schema('hep')
+    subschema = schema['properties']['arxiv_eprints']
+
+    filename = pkg_resources.resource_filename(
+        __name__, os.path.join('fixtures', '1711.10662.tar.gz'))
+
+    data = {
+        'arxiv_eprints': [
+            {
+                'categories': [
+                    'cs.CV',
+                ],
+                'value': '1711.10662',
+            },
+        ],
+    }  # holdingpen/807096
+    extra_data = {}
+    files = MockFiles({
+        '1711.10662.tar.gz': AttrDict({
+            'file': AttrDict({
+                'uri': filename,
+            }),
+        }),
+    })
+    assert validate(data['arxiv_eprints'], subschema) is None
+
+    obj = MockObj(data, extra_data, files=files)
+    eng = MockEng()
+
+    try:
+        temporary_dir = mkdtemp()
```

Comment:
You can use pytests tmpdir: https://docs.pytest.org/en/latest/tmpdir.html

Reply:
That fixture is a bit weird. It creates in the temporary directory a folder structure like `pytest-of-jmartinm/pytest-1/test_arxiv_plot_extract_handle0` and then it fails with  `EEXIST: [File exists]:` when creating a file...